notebooks/bimodal_dist_plot.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Another example, with two modes
</pre><p>Reuse entire code by creating a function. Check code of Figure 2.1 for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>